interfaces/repo.go

```diff
@@ -741,6 +741,9 @@ func (r *Repository) AutoConnectBlacklist(snapName string) map[string]bool {
 
 	for plugName, plug := range r.plugs[snapName] {
 		iface := r.ifaces[plug.Interface]
+		// XXX: what do about this? it's mostly an optimisation,
+		// anyway this is logic is probably already doing the wrong
+		// thing in some corner cases
```

Comment:
If this isn't going to be fixed by the of this series of PRs, can someone file a bug?